meetuppizza/tests.py

```diff
@@ -35,6 +35,11 @@ def test_user_is_created(self):
     user = User.objects.get(username='Bjorn')
     self.assertFalse(user == None)
 
+  def test_super_user_is_created(self):
+    self.client.post('/sign_up', self.params)
+    user = User.objects.get(username='adminotaur')
+    self.assertFalse(user == None)
```

Comment:
Think about what this assertion is actually testing. Is this enough to verify that `user` is a superuser? It's always useful to check your test method names against your assertions when you're done writing a test. If they don't match up, you may need to rename the test, or add an additional one.

In this case, you're really testing that a user named `adminotaur` exists, but you're not really testing anything about whether that user is a superuser or not. Starting with existence is the first step if you're TDDing this, but it looks like there are a few missing tests here.

I left a similar comment over here: https://github.com/nicole-a-tesla/meetup.pizza/pull/9/files#r55149526.